src/app/[locale]/game/[id]/page.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import { useTranslations } from 'next-intl';
+import { useQuery } from '@tanstack/react-query';
+import Loader from '@icons/Loader.svg';
+import fetchGame from '@/apiUrls/game';
+import queryKeys from '@/constants/queryKeys';
+import ErrorWrapper from '@/components/errorWrapper';
+
+interface IGamePage {
+  params: { id: number | string };
+}
+
+function GamePage({ params }: IGamePage) {
+  const t = useTranslations();
+
+  const gameId = params.id || '';
+
+  const { isLoading, isError, data } = useQuery({
+    queryKey: queryKeys.fetchGame,
+    queryFn: () => fetchGame(gameId),
+    refetchOnWindowFocus: false,
+    gcTime: 0,
```

Comment:
tu możemy jeszcze dodać zabezpieczenie enabled: Boolean(gameId), żeby odpalać to query tylko jeśli gameId istnieje